web/src/components/apps/Dashboard.tsx

```diff
@@ -749,11 +751,24 @@ class Dashboard extends Component<Props, State> {
 
     return (
       <>
-        {!app && (
-          <div className="flex-column flex1 alignItems--center justifyContent--center">
-            <Loader size="60" />
-          </div>
-        )}
+        {!app ||
+          (this.state.slowLoader && (
+            <div
```

Comment:
added slowLoader state, but this is always false for now. 
changed some of the styles so it only shows loader on the dashboard not the multi app sidebar